contracts/src/v0.8/dev/functions/FunctionsOracle.sol

```diff
@@ -165,7 +165,7 @@ contract FunctionsOracle is
       gasPrice
     );
     uint96 requiredFee = getRequiredFee(data, billing);
```

Comment:
Can we call this oracleFee or donFee?

Reply:
Done. Good suggestion, it's a lot more clear.